### DISCUSSION TITLE
Monthly Update - Aug 2024

### DISCUSSION THREAD

**Discussion by boundless-forest** [ringdao-community-guild/discussion#174] [Show and tell]
# **Protocol Updates**

- Enable async backing on the Crab/Darwinia networks. https://github.com/darwinia-network/darwinia/pull/1554
- Delete the EthereumXCM pallet. https://github.com/darwinia-network/darwinia/pull/1553
- xAPI Aggregator. https://github.com/xapi-box/xapi-contracts
- DIP7 testing & audit & update. https://github.com/darwinia-network/darwinia/pull/1543
- Update deposit precompile interfaces. https://github.com/darwinia-network/darwinia/pull/1561

# Darwinia Ecosystem Applications and Infrastructure **Updates**

- Darwinia Bootcamp lesson 1, 2.
- RingDAO based on the AragonDAO. https://github.com/darwinia-network/vote.ringdao.com
- Msgport docs: add integrate guide. https://github.com/msgport/docs/pull/48
- KtonDAO staking app new design. https://staking.ktondao.xyz/
- Paralink new design. https://github.com/darwinia-network/paralink-ui

**Comment by Laki389** [comment]
Hi @boundless-forest 
please don't forget to post these monthly reports on Darwinia Medium as well https://medium.com/darwinianetwork

Thank you. :+1: :pray:

**Reply by boundless-forest** [reply]
https://medium.com/darwinianetwork/monthly-update-aug-2024-d35c59a5d9f7 done.

**Comment by nblogist** [comment]
Let's discuss on planning videos more related to the protocol updates on today's call!
